package.json

```diff
@@ -185,7 +187,7 @@
         },
         {
           "command": "vscode-docker.createWebApp",
-          "when": "view == dockerExplorer && viewItem =~ /^(azureImageNode|dockerHubImageTag)$/"
+          "when": "view == dockerExplorer && viewItem =~ /^(azureImageTagNode|dockerHubImageTag|customImageTag)$/"
```

Comment:
nit: I'd prefer something like `/^(azure|dockerHub|custom)ImageTag/$` (NOTE: You'd have to remove `Node` off of the Azure one or add it to the other ones to be consistent)

Reply:
I'd rather not do it that way because it makes them harder to search for and also assumes something about their structure.  Did take the time to make these more consistent, though.